packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -1,24 +1,33 @@
 import { useFeature } from '../components/GrowthBookProvider';
 import { feature } from '../lib/featureManagement';
 import { FeedLayout } from '../lib/featureValues';
+import { AllFeedPages } from '../lib/query';
+import { SharedFeedPage } from '../components/utilities';
 
 interface UseFeedLayoutProps {
-  feedName: string;
+  feedName: AllFeedPages;
+}
+
+interface UseFeedLayoutReturn {
+  isSingleColumnFeedLayout: boolean;
 }
 
-// TODO: return type to be changed once more info is added to the return object
 export const useFeedLayout = ({
   feedName,
-}: UseFeedLayoutProps): Record<string, unknown> => {
+}: UseFeedLayoutProps): UseFeedLayoutReturn => {
   const feedLayoutVersion = useFeature(feature.feedLayout);
   const isFeedLayoutVersion = feedLayoutVersion === FeedLayout.V1;
-  const shouldUseFeedLayout = isFeedLayoutVersion && feedName !== 'squad';
+  const isIncludedFeed = Object.values(SharedFeedPage).includes(
```

Comment:
We did say squad page would be excluded only because of the header right?
Just trying to remember why we didn't apply it there.

Reply:
So we are only applying it to shared feed pages https://dailydotdev.slack.com/archives/C061T644XV3/p1700218672945989 based on this discussion.

So basically there could be more edge cases there and product wants to experiment with these feeds first..